### DISCUSSION TITLE
Do you have any job vacancies for front end ?

### DISCUSSION THREAD

**Discussion by LuizinhoVI** [community/community#53817] [Programming Help] [answered]
### Body

I'm studying and developing projects every day, but in my city it's small, there's not much opportunity in the vacancy, does anyone know any company or person that is hiring dev js, c++, css, html, jquery, json, sql and I'm studying .net

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by hrishikesh-kocharekar-dev** [comment, marked as answer]
Hey Luiz! I noticed that you mentioned you're from a small city. Have you thought about applying for remote 🌏 jobs? If you're interested in a front-end development role, I've seen some companies hiring developers who only know HTML and CSS, and not even JavaScript - just plain HTML and CSS. You can easily learn JavaScript frameworks such as React or Svelte; Svelte is even easier to learn than React. By learning these frameworks, you can increase your chances of getting hired as a remote front-end developer.

**Reply by LuizinhoVI** [reply]
> Oi Luiz! Percebi que você mencionou que é de uma cidade pequena. Você já pensou em se candidatar a remoto🌏empregos? Se você estiver interessado em uma função de desenvolvimento front-end, já vi algumas empresas contratando desenvolvedores que só conhecem HTML e CSS, e nem mesmo JavaScript - apenas HTML e CSS simples. Você pode facilmente aprender frameworks JavaScript como React ou Svelte; Svelte é ainda mais fácil de aprender do que React. Ao aprender essas estruturas, você pode aumentar suas chances de ser contratado como desenvolvedor front-end remoto.

thank you very much  :)

**Comment by sumitshinde-84** [comment]
Hello,
My name is Sumit Shinde, and I am a passionate and hardworking full-stack developer. I have recently completed my education and am actively seeking job opportunities to kickstart my career. While I don't have any professional experience yet, I have gained valuable skills and knowledge through my education and by contributing to open-source projects.
